ui/apps/platform/src/ConsolePlugin/SecurityVulnerabilitiesPage/Index.tsx

```diff
@@ -2,20 +2,26 @@ import * as React from 'react';
 import { PageSection, Title } from '@patternfly/react-core';
 import { CheckCircleIcon } from '@patternfly/react-icons';
 
-import SeverityCountLabels from 'Containers/Vulnerabilities/components/SeverityCountLabels';
+import SummaryCounts from 'Containers/Dashboard/SummaryCounts';
+import ViolationsByPolicyCategory from 'Containers/Dashboard/Widgets/ViolationsByPolicyCategory';
+import PluginProvider from '../PluginProvider';
 
 export function Index() {
     return (
-        <>
+        <PluginProvider>
             <PageSection>
                 <Title headingLevel="h1">{'Hello, Plugin!'}</Title>
-                <SeverityCountLabels
-                    criticalCount={10}
-                    importantCount={20}
-                    moderateCount={30}
-                    lowCount={40}
-                    unknownCount={50}
+                <SummaryCounts
+                    hasReadAccessForResource={{
+                        Cluster: true,
+                        Node: true,
+                        Alert: true,
+                        Deployment: true,
+                        Image: true,
+                        Secret: true,
+                    }}
                 />
+                <ViolationsByPolicyCategory />
```

Comment:
Two placeholder components, one that makes a GET REST request and one that makes a POST GraphQL request.